tests/end_to_end/test_yara_integration/test_yara_integration.py

```diff
@@ -0,0 +1,84 @@
+import os
+import json
+import re
+import pytest
+from datetime import datetime
+from tempfile import gettempdir
+
+from wazuh_testing.tools.time import parse_date_time_format
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+
+# Test cases data
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_path = os.path.join(test_data_path, 'test_cases')
+test_cases_file_path = os.path.join(test_cases_path, 'cases_yara_integration.yaml')
+yara_script = os.path.join(test_data_path, 'configuration', 'yara.sh')
+malware_downloader_script = os.path.join(test_data_path, 'configuration', 'malware_downloader.sh')
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+
+# Playbooks
+configuration_playbooks = ['configuration.yaml']
+events_playbooks = ['generate_events.yaml']
+teardown_playbooks = ['teardown.yaml']
+configuration_extra_vars = {'yara_script': yara_script, 'malware_downloader_script': malware_downloader_script}
+
+# Configuration
+configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
+@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+def test_yara_integration(configure_environment, metadata, get_dashboard_credentials, generate_events,
+                          clean_alerts_index):
+    rule_description = metadata['rule.description']
+    rule_id = metadata['rule.id']
+    rule_level = metadata['rule.level']
+    data_yara_rule = metadata['extra']['data.yara_rule']
+    timestamp_regex = r'\d+-\d+-\d+T\d+:\d+:\d+\.\d+[\+|-]\d+'
+
+    expected_alert_json = fr".+timestamp\":\"({timestamp_regex})\",.+level\":{rule_level}.+description\"" \
+                          fr":\"{rule_description}.+id.+{rule_id}"
+
+    expected_indexed_alert = fr".+yara_rule\": \"{data_yara_rule}.+level.+{rule_level}.+id.+{rule_id}.+" \
+                             fr"timestamp\": \"({timestamp_regex})\""
+
+    query = e2e.make_query([
+        {
+          "term": {
+            "rule.id": f"{rule_id}"
+          }
+        },
+        {
+          "term": {
+            "rule.level": f"{rule_level}"
+          }
+        },
+        {
+          "term": {
+            "data.yara_rule": f"{data_yara_rule}"
+          }
+        }
+    ])
```

Comment:
Get the alert with the timestamp. You can check other tests in the branch.

Reply:
Done in [df2a57b](https://github.com/wazuh/wazuh-qa/pull/3115/commits/df2a57b36e82d86c756ab4da90e286cc955d3799)